x-pack/plugins/cases/server/client/metrics/lifespan.test.ts

```diff
@@ -120,7 +120,7 @@ describe('lifespan', () => {
         [
           {
             attributes: { payload: { hello: 1, status: CaseStatuses.closed }, type: 'status' },
-          } as unknown as SavedObject<CaseUserActionInjectedAttributes>,
+          } as unknown as SavedObject<UserActionAttributesWithInjectedId>,
```

Comment:
Maybe we can create a type `UserActionTransformedAttributes` and put it in `x-pack/plugins/cases/server/common/types/user_actions.ts` as we did with `CaseSavedObjectTransformed` (`x-pack/plugins/cases/server/common/types/case.ts`). I think it is nice to be consistent with our naming. Wdyt?

Reply:
I'm going to leave these as is for now since they're in the client.